### PR TITLE
Merge of "basic alignment" PR from develop to 1.12 branch

### PR DESCRIPTION
**Pull request by qkoziol**
Merge of https://github.com/HDFGroup/hdf5/pull/115 to the 1.12 branch.

### CODE REVIEW

File: src/H5F.c [hunk 1]
```diff
@@ -1049,25 +1041,25 @@ H5Fget_file_image(hid_t file_id, void *buf_ptr, size_t buf_len)
...
 
     FUNC_ENTER_API(FAIL)
-    H5TRACE2("e", "i*x", file_id, config_ptr);
+    H5TRACE2("e", "i*x", file_id, config);
 
     /* Check args */
-    if ((NULL == config_ptr) || (config_ptr->version != H5AC__CURR_CACHE_CONFIG_VERSION))
-        HGOTO_ERROR(H5E_ARGS, H5E_BADVALUE, FAIL, "Bad config_ptr")
+    if ((NULL == config) || (config->version != H5AC__CURR_CACHE_CONFIG_VERSION))
+        HGOTO_ERROR(H5E_ARGS, H5E_BADVALUE, FAIL, "Bad configptr")
```
Comment (derobins) [inline src/H5F.c:1054]: Bad search-and-replace?

**Review by derobins — APPROVED** [review]

**Comment by lrknox** [comment]
Why have checks for this PR not reported back after 2 days?

File: src/H5F.c (hunk 1 above)
Comment (qkoziol) [reply src/H5F.c:1054]: Yeah, I fixed it in the next "cleanup" merge: https://github.com/HDFGroup/hdf5/pull/132

**Comment by qkoziol** [comment]
> Why have checks for this PR not reported back after 2 days?

Weird - can I re-trigger them?  (Without pushing a change to the PR)

**Comment by lrknox** [comment]
> > Why have checks for this PR not reported back after 2 days?
> 
> Weird - can I re-trigger them? (Without pushing a change to the PR)

There is a "Rerun jobs" button on the "Checks" page, in this case https://github.com/HDFGroup/hdf5/pull/137/checks.   I clicked it, and the Formatting check reported, but the others are still waiting.

We can override the requirement and merge anyway, but I would like to know why this PR is the only one where we've seen this.

**Comment by qkoziol** [comment]
> > > Why have checks for this PR not reported back after 2 days?
> > 
> > 
> > Weird - can I re-trigger them? (Without pushing a change to the PR)
> 
> There is a "Rerun jobs" button on the "Checks" page, in this case https://github.com/HDFGroup/hdf5/pull/137/checks. I clicked it, and the Formatting check reported, but the others are still waiting.
> 
> We can override the requirement and merge anyway, but I would like to know why this PR is the only one where we've seen this.

Maybe merge this PR and create an issue for the CI check weirdnes?   It would be nice to get this one merged, so I can move forward with the next batch of merges back to 1.12.

**Comment by byrnHDF** [comment]
Maybe because it is coming in from a branch, which is not in the list? Format check does not check for a branch.

If I remember correctly that was the wish of the developers for the fork commits. Maybe the main.yml should be changed to all branches?

Allen

________________________________________

File: src/H5VLcallback.c [hunk 2]
```diff
@@ -29,7 +29,7 @@
 
 #include "H5private.h"   /* Generic Functions                                */
 #include "H5Eprivate.h"  /* Error handling                                   */
-#include "H5Fprivate.h"  /* File access				            */
+#include "H5Fprivate.h"  /* File access				             */
```
Comment (byrnHDF) [inline src/H5VLcallback.c:32]: Maybe remove space instead of adding a space?

**Review by byrnHDF — APPROVED** [review]

**Comment by qkoziol** [comment]
Looks like someone with more permissions that I have will need to squash merge this.

**Pull request merged**